pkg/cmd/attestation/verify/verify.go

```diff
@@ -244,23 +246,21 @@ func runVerify(opts *Options) error {
 	}
 
 	// Apply predicate type filter to returned attestations
-	if opts.PredicateType != "" {
-		filteredAttestations := verification.FilterAttestations(opts.PredicateType, attestations)
-
-		if len(filteredAttestations) == 0 {
-			opts.Logger.Printf(opts.Logger.ColorScheme.Red("✗ No attestations found with predicate type: %s\n"), opts.PredicateType)
-			return err
-		}
-
-		attestations = filteredAttestations
+	filteredAttestations := verification.FilterAttestations(opts.PredicateType, attestations)
```

Comment:
I agree this can be reorganized a bit. I was thinking of revisiting it in a follow up. I think I'm going to be doing reorganizing soon when I plan to print more information about the policy the command is verifying, was thinking of addressing this at that point. But I can address it sooner.